### PR TITLE
Troll updates, fixed delay

### PR DESCRIPTION
**Pull request by ducky**

### CODE REVIEW

File: lib/utils/format.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export default (function() {
```
Comment (thezanke) [inline lib/utils/format.js]: You don't need to export anything here, or even use a self-executing function. Simply importing into lib/giantbot.js will run anything in the file.

File: lib/utils/format.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  String.prototype.format = function () {
```
Comment (thezanke) [inline lib/utils/format.js]: Extending native prototypes makes my linter explode... I can either add an exception for "String" to the ESLint config, or we can make this an actual utility function where you'd call it on a string, whichever you prefer.

File: lib/utils/format.js (hunk 1 above)
Comment (ducky) [reply lib/utils/format.js]: I figured, wasn't sure exactly how import/export worked in this case.

File: lib/utils/format.js (hunk 2 above)
Comment (ducky) [reply lib/utils/format.js]: It doesn't matter to me. The extension is easier to use, but I know it's frowned upon. Doesn't matter to me either way.

File: lib/skills/trolls.js [hunk 3]
```diff
@@ -94,33 +109,33 @@ const skill = ({ controller }) => {
...
     // Lower user count if they stop typing before reaching threshold
     } else {
-      resetTimer(bot, message, userID, typistEntry, Math.round(typistEntry.delay / 3));
+      resetTimer(bot, message, user, typistEntry, RESET_DELAY);
     }
   });
 
   // Check if user sent message
-  controller.on('ambient', function(bot, message) {
+  controller.on('ambient', (bot, message) => {
```
Comment (thezanke) [inline lib/skills/trolls.js:110]: You da man

Comment (ducky) [reply lib/skills/trolls.js:110]: I didn't realize I had functions here. I copy pasta'd the example from BotKit originally, lol.

File: lib/utils/format.js (hunk 1 above)
Comment (thezanke) [reply lib/utils/format.js]: Get rid of the export here and I'll merge, I'll add the exception to ESLint to allow string prototype extensions

**Pull request merged**